.github/actions/load-img/action.yml

```
@@ -23,18 +27,29 @@ runs:
     - name: Install `@actions/artifact`
       shell: bash
       run: |
-        pnpm install @actions/artifact@1 -w
+        pnpm install @actions/artifact -w
```

Shouldn't this be fixed to version 2? When the next breaking version get released it can break our workflows.
```suggestion
        pnpm install @actions/artifact@2 -w
```

---

Gotcha, granted!